### PR TITLE
Added antipatterns & new rule

### PR DESCRIPTION
**Pull request by vkyfox**
Added antipatterns for ACCORD_NOMBRE #1057
Added rule TOUT_A_COMMENCE #1080

### CODE REVIEW

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 1]
```diff
@@ -8193,18 +8193,41 @@ though, and has since been slightly modified)-->
...
+    </antipattern>
+    <antipattern>
+        <token regexp="yes">\d+</token>
+        <token regexp="yes">éditions</token>
+    </antipattern>
+    <antipattern>
+        <token regexp="yes">Windows|Apple|Galaxy</token>
+        <token regexp="yes">\d+</token>
+        <token postag=".*s"/>
+    </antipattern>
```
Comment (dpelle) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:8213]: I suggest to add examples.
They will help to understand why we have antipattern and the will be useful for regression testing.

Comment (vkyfox) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:8213]: Understood, I'll remember to add examples in the future.

**Comment by tiff** [comment]
@dpelle could we merge this?

**Comment by danielnaber** [comment]
I've merged it now. @dpelle let me know in case that wasn't okay.

**Pull request merged**